xonsh/xoreutils/util.py

```diff
@@ -0,0 +1,15 @@
+def arg_handler(args, out, short, key, val, long=None):
```

Comment:
well, the most common convention is to call helper modules `utils.py` (mind the plural).